Makefile

```diff
@@ -6,6 +6,7 @@ all: clean build install
 .PHONY: fmt
 fmt:
 	go fmt ./...
+	goimports ./..
```

Comment:
I don't think goimports belongs in the makefile (have you tried running it?)
 It's more a tool to configure with your editor.

https://godoc.org/golang.org/x/tools/cmd/goimports